.circleci/config.yml

```diff
@@ -0,0 +1,156 @@
+version: 2.1
+
+aliases:
+  - &working_directory ~/ui-kit
+
+  - &restore_yarn_cache
+    name: 'Restoring yarn cache'
+    keys:
+      - v1-yarn-cache-{{ checksum "yarn.lock" }}
+      - v1-yarn-cache
+
+  - &save_yarn_cache
+    name: 'Saving yarn cache'
+    key: v1-yarn-cache-{{ checksum "yarn.lock" }}
+    paths:
+      - ~/.cache/yarn
```

Comment:
```suggestion
      - ~/.cache/yarn
      - node_modules
```

Reply:
Won't add. Isn't mentioned [here](https://circleci.com/docs/2.0/yarn/). Would prefer to just store the yarn cache and not installed files.

Reply:
We do this in all our circleci repos!

https://github.com/commercetools/merchant-center-frontend/pull/4303